docs/MAAP_USAGE.md

```diff
@@ -287,7 +284,6 @@ inputs = dict(
    beams="all",
    columns="agbd, agbd_se, sensitivity, geolocation/sensitivity_a2",
    query="l2_quality_flag == 1 and l4_quality_flag == 1 and sensitivity > 0.95 and `geolocation/sensitivity_a2` > 0.95",
-   limit=0,
```

Comment:
Because we're using the algorithm's registered default, we no longer need to specify the limit as 0 in order to use the default limit? Do we still need DEFAULT_LIMIT?

Reply:
We don't have to provide the value when we want to get the default. I had originally put `limit=0` in there because I didn't fully understand how DPS passes inputs through to the "run" script. That was why I also thought that for string inputs we needed to pass `"-"` to represent using the default. Bottom line, we don't need to specify a "dummy" value for inputs where we want the default.

Reply:
for clarification, the default is being registered inside of the algorithm_config or the default is still being set by DEFAULT_LIMIT alongside the `limit` input parameter for the main function?

Reply:
If not supplied, `limit` is set to `DEFAULT_LIMIT`:

```python
    limit: int = typer.Option(
        DEFAULT_LIMIT,
        callback=lambda value: DEFAULT_LIMIT if value < 1 else value,
        help="Maximum number of granules to subset",
    ),
```